tokens/sass-helpers.js

```diff
@@ -7,21 +7,24 @@ const chroma = require('chroma-js');
  * based on contrast value of the input color
  *
  * @param color - chroma-js color instance
- * @param {String} [light] - light color variant, defaults to 'yiq-text-light' from ./src/global/other.json
- * @param {String} [dark] - dark color variant, defaults to 'yiq-text-dark' from ./src/global/other.json
+ * @param {String} [themeVariant] - theme variant name that will be used to find default contrast colors
+ * @param {String} [light] - light color variant, defaults to 'yiq-text-light'
+ * from ./src/themes/{themeVariant}/other.json
+ * @param {String} [dark] - dark color variant, defaults to 'yiq-text-dark'
+ * from ./src/themes/{themeVariant}/other.json
```

Comment:
[clarification] I don't necessarily think the theme variants are necessarily defaulting to `yiq-text-light` or `yiq-text-dark` depending on the theme variant itself. That choice depends on the `color` being modified (yiq-ified? 😅).

Should the JSDoc comment here just reference that the `light` theme pulls both yiq light/dark values from the `other.json` file in `light` or `dark` folder, respectively, without an explicit mention of `yiq-text-light` or `yiq-text-dark` here?